interface/wc_event_pictures.gfx

```diff
@@ -594,4 +594,8 @@ spriteTypes = {
 		texturefile = "gfx\\event_pictures\\gorian_empire.tga"
 		allwaystransparent = yes
 	}
+	spriteType = {
+		name = "GFX_evt_kobold_statute"
```

Comment:
We already have this picture, `GFX_evt_cathedral_cult_of_the_candle`.
Plus, never stretch the pictures.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/83752214-334a9180-a679-11ea-86e2-04c5553bcad4.png)

</details>